velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -57,6 +57,79 @@ class DateTimeFunctionsTest : public SparkFunctionBaseTest {
   }
 };
 
+TEST_F(DateTimeFunctionsTest, toUTCTimestamp) {
+  const auto toUTCTimestamp = [&](const std::string_view ts,
+                                  const std::optional<std::string> tz) {
+    auto timestamp = std::make_optional<Timestamp>(
+        util::fromTimestampString(ts.data(), ts.length()));
+    return evaluateOnce<Timestamp>("to_utc_timestamp(c0, c1)", timestamp, tz);
+  };
+  EXPECT_EQ(
+      util::fromTimestampString("2015-07-24 07:00:00"),
```

Comment:
nit: consider changing lambda to return a string representation of the timestamp; this would allow to shorten the tests and make them more readable

```
EXPECT_EQ("2015-07-24 07:00:00", toUtcTimestamp(...));
```

Reply:
I don't think there's a function that does Timestamp -> string. I could add the Timestamp representation in the LHS of the EXPECT_EQ?

Reply:
tmToString in velox/type/Timestamp.h ?